inception-resnet-v2/node.html

```diff
@@ -0,0 +1,183 @@
+        <input type="text" id="node-input-service">
+    </div>
+
+    <div class="form-row">
+        <label for="node-input-method"><i class="icon-tasks"></i> <span data-i18n="ModelAssetExchangeServer.label.method"></span></label>
+        <select id="node-input-method">
+            <option value="get_metadata" data-i18n="ModelAssetExchangeServer.parameters.get_metadata"></option>
+            <option value="predict" data-i18n="ModelAssetExchangeServer.parameters.predict"></option>
+        </select>
+        &nbsp;
+        <input type="checkbox" id="optional-parameters" style="margin-left: 10px; vertical-align: text-bottom; width: auto;">
+        <label for="optional-parameters" id="optional-parameters-label" style="width: auto;"> <span data-i18n="ModelAssetExchangeServer.parameters.optionalParameters"></span></label>
+    </div>
+
+    <div class="form-row" id="predict_image">
+        <label for="node-input-predict_image"><i class="fa fa-list"></i> <span data-i18n="ModelAssetExchangeServer.parameters.image"></span></label>
+
+        <input type="text" id="node-input-predict_image" placeholder="" style="width:70%">
+        <input type="hidden" id="node-input-predict_imageType">
+    </div>
+
+    <hr/>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="node-red:common.label.name"></span></label>
+        <input type="text" id="node-input-name" data-i18n="[placeholder]node-red:common.label.name">
+    </div>
+</script>
+
+<script type="text/x-red" data-help-name="inception-resnet-v2">
+
+    <p>This model recognizes the 1000 different classes of objects in the ImageNet 2012 Large Scale Visual Recognition Challenge. The model consists of a deep convolutional net using the Inception-ResNet-v2 architecture that was trained on the ImageNet-2012 data set. The input to the model is a 299×299 image, and the output is a list of estimated class probabilities.</p>
+    <hr>
+    <h1>predict</h1>
+    <h3>Inputs:</h3>
+    <dl class="message-properties">
+        <dt>payload <span class="property-type">Buffer</span></dt>
+        <dd>Buffer data of PNG or JPEG image.</dd>
+    </dl>
+    <h3>Outputs:</h3>
+    <dl class="message-properties">
+        <dt>payload <span class="property-type">object</span></dt>
+        <dd>Prediction</dd>
+    </dl>
+    <br>
+    <h1>metadata</h1>
+    <h3>Inputs:</h3>
+    <dl class="message-properties">
+        <dt>none <span class="property-type">none</span></dt>
+    </dl>
+    <h3>Outputs:</h3>
+    <dl class="message-properties">
+        <dt>payload <span class="property-type">object</span></dt>
+        <dd>Returns the metadata associated with the model.</dd>
+    </dl>
+
+</script>
+<script type="text/javascript">
+    RED.nodes.registerType('inception-resnet-v2-service', {
+        category: 'config',
+        defaults: {
+            host: { value: "http://max-inception-resnet-v2.max.us-south.containers.appdomain.cloud", required: true },
```

Comment:
@ptitzler In terms of data encryption, I think that `https://max-inception-resnet-v2.max.us-south.containers.appdomain.cloud` is better than the URL which starts with `http://`.

Reply:
I changed all `http` references to `https`.